python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -713,9 +714,7 @@ def len_sources(apdb_query):
         Number of diaSources in this APDB.
     """
     with apdb_query.connection as connection:
-        cursor = connection.cursor()
-        cursor.execute('select count(*) FROM "DiaSource";')
-        count = cursor.fetchone()[0]
+        count = connection.execute('select count(*) FROM "DiaSource";').first()[0]
```

Comment:
There is also a `scalar()` shortcut which does (almost) the same as `first()[0]`.

Reply:
I want `first` here, so that it raises if no results are found, which could only be because there's something wrong with the database.

Reply:
If something is wrong with the database then `execute()` will raise an exception. If it does not raise then `count(*)` is guaranteed to return exactly one row and one column.